### ISSUE TITLE
How to make a shared Angular Web + NativeScript Angular Mobile app?

### ISSUE THREAD

**Issue by Rob4226** [NativeScript/nx#30]
What is the proper procedure to start a new nx workspace for a shared NativeScript Angular Mobile App and Angular Web App?

Right now I have a NativeScript 7 / Angular 11 shared mobile/web app that uses the .tns.ts, .tns.html, .web.ts, etc... method of code splitting, but it seems the new NativeScript 8 and @nativescript/webpack do not support that anymore.  This is a substantial project and I want to upgrade to NS8 /Angular 12 so I guess I need to convert the project to work in an nx workspace using the `libs` folder for the shared Angular modules.  But I want to make sure I set up the nx workspace properly first before taking on this massive undertaking.

**So basically, can someone please outline the proper commands to start a shared mobile/web NativeScript 8 / Angular 12 project in an nx workspace?**

I tried staring an Angular nx workspace project then adding NativeScript but the nx commands to run the mobile project would not work, I kept getting `Unable to install dependencies.`.  It made a minimal nested package.json in the mobile project with, `@nativescript/core`, `@nativescript/android`, and `@nativescript/ios` and the rest of the dependencies in the root package.json which seems right but idk.   Any help would greatly be appreciated.  Thank you!!

**Comment by lostation** [comment]
[Im fighting with that topic here](https://github.com/nstudio/xplat/issues/275)...goog luck !
